src/shared.ts

```diff
@@ -1,66 +1,112 @@
-type FontWeight = 'normal' | 'bold' | '100' | '200' | '300' | '400' | '500' | '600' | '700' | '800' | '900';
+import { Platform } from 'react-native';
 
-const defaultFontConfig = {
-    regular: {
+export const fontConfig = {
+    customVariant: {
+        fontFamily: Platform.select({
+            web: 'Roboto, "Helvetica Neue", Helvetica, Arial, sans-serif',
+            ios: 'System',
+            default: 'OpenSans-Regular',
+        }),
+        fontWeight: '400' as const,
+        letterSpacing: 0.5,
+        lineHeight: 22,
+        fontSize: 20,
+    },
```

Comment:
what does "customVariant" do?

Reply:
It will apply platform specific font styles in which the application is running.

Reply:
So... I have 2 questions,
* when will our adopters be using this `customVariant` thing, and how?
* Why are "Roboto" and "System" the default font family here?

Reply:
I am not sure on this, @joebochill Could you please give your valuable input here.

Reply:
I didn't write this code, I'm not sure why you're asking me to explain it.